src/pages/[userId]/prospects.js

```diff
@@ -108,39 +89,47 @@ export default function Prospects() {
           <h4>Other Ministers</h4>
         </Grid>
       </Grid>
-      {prospects.map((prospect) => (
-        <Grid
-          container
-          spacing={1}
-          key={prospect.id}
-          className="striped columns"
-        >
-          <Grid item md={2} xs={3}>
-            <p>{path(['names', '0'], prospect)}</p>
-          </Grid>
-          <Grid item md={2} xs={3}>
-            <p>{prop('status', prospect)}</p>
-          </Grid>
-          <Grid item md={2} xs={3}>
-            <p>{prop('last_contacted', prospect)}</p>
-          </Grid>
-          <DesktopOnly>
-            <Grid item md={3}>
-              <p>{path(['notes', '0'], prospect)}</p>
+      {isLoadingProspects && <h1>Loading data...</h1>}
+      {prospects
+        && prospects.map((prospect) => (
+          <Grid
+            container
+            spacing={1}
+            key={prospect.id}
+            className="striped columns"
+          >
+            <Grid item md={2} xs={3}>
+              <p>{getPreferredName(prospect)}</p>
+            </Grid>
+            <Grid item md={2} xs={3}>
+              <p>{prop('status', prospect)}</p>
+            </Grid>
+            <Grid item md={2} xs={3}>
+              <p>
+                {/* Days are showing a day behind what is in the DB. Adding a day to resolve it. */}
```

Comment:
Looks like the `eo` token is for "local day of week" and you probably want to use this instead:
![image](https://user-images.githubusercontent.com/38568524/135727871-3350c87b-858f-400f-bd73-f66551510890.png)

Reply:
Seems like it's still not getting the day correctly. I suspect this wouldn't be as big of a problem if we stored the date in a more correct format. In the long term I'd like to actually store a standardized date. I think I was just having trouble generating the fake data that's in the db right now so ended up just going with these simplified date strings.

Reply:
This will get you what you need:
```
{format(
  parse(
    prop('last_contacted', prospect),
    'yyyy-MM-dd',
    new Date(),
  ),
  'MMM do',
)}
```
And btw with that import for `add` from `date-fns` you could have just included it in the curly braces with `format`. You can do the same with `parse`.

Reply:
That makes sense. I fixed the importing and added your formatting code. I guess the other one was 0 based like normal Date in javascript and the local day of the week is fixed?

Reply:
I suspect it was something about how the `Date` constructor was parsing the string.